profiles/ldap.py

```diff
@@ -252,7 +252,19 @@ def ldap_update_profile(form_input, uid):
             form_input[key] = None
 
     if not form_input["name"] == account.cn:
-        account.cn = form_input["name"]
+        full_name = form_input["name"]
+        account.cn = full_name
+        account.gecos = full_name
+        account.givenname = full_name.split()[0]
+        if len(full_name.split()) <=1:
+            account.sn = full_name
+        else:
+            account.sn = "".join(full_name.split().pop(0))
```

Comment:
What about something along the lines of 
```suggestion
        account.sn = full_name.removeprefix(account.givenname).strip()
```

This would remove the first part of the name, but leave it blank if there are no other parts.

Reply:
`sn` is a required field in our ldap configuration, so as far as I know I can't leave it blank. However, that is a cleaner method of what I'm trying to do in the `else` statement, so I will implement it there.

Reply:
Oh, it can't be an empty string? How about we test that before we look to merge this? I should be able to tonight.